game/armorGroupLocation.py

```diff
@@ -0,0 +1,7 @@
+from game.point_with_heading import PointWithHeading
+
+
+class ArmorGroupLocation:
+    def __init__(self, PointWithHeading, bool):
```

Comment:
Wrong naming convention. `point_with_heading`.

Also, don't name a variable "bool". That shadows the built in type and tells us nothing about its purpose.

Finally, your arguments need types (interestingly it looks like you actually tried to give them types but forgot to give them _names_, which means that they just have names that shadow types without having any types themselves).